.github/workflows/build-centos7-assets.yaml

```diff
@@ -0,0 +1,44 @@
+name: Build CentOS 7 Buildbox Assets Images
+run-name: Build CentOS 7 Buildbox Assets Images
+on:
+  # Only allow manual triggers
+  workflow_dispatch:
+
+env:
+  REGISTRY: ghcr.io
+  BUILDBOX_BASE_NAME: ghcr.io/gravitational/teleport-buildbox
```

Comment:
I know this isn't you, but curious if we can just change `BUILDBOX_BASE_NAME` in `build.assets/images.mk`. Do we even use the ECR buildbox anymore?

Reply:
Drone still pushes to ECR. I also think we still use that as our "official images" https://goteleport.com/docs/management/guides/docker/